include/fmt/xchar.h

```diff
@@ -127,6 +127,18 @@ auto join(const std::tuple<T...>& tuple, basic_string_view<wchar_t> sep)
   return {tuple, sep};
 }
 
+template <typename... T>
+auto format(wformat_string<T...> fmt, T&&... args) -> std::wstring {
+  return vformat(fmt::wstring_view(fmt), fmt::make_wformat_args(args...));
+}
```

Comment:
There is no need to move this function definition, please revert.

Reply:
I've reverted the move.

This patch only fixes format_to called with a compile time string (FMT_STRING). Looks like the other functions calling detail::to_string_view in this file has the same problem, like `vformat_to`, `vformat` and maybe `formatted_size`...

Reply:
`vformat*` are fine because they take runtime format strings.